src/solc_0.8/assetERC721/BaseERC721.sol

```diff
@@ -0,0 +1,150 @@
+//SPDX-License-Identifier: MIT
+// solhint-disable-next-line compiler-version
+pragma solidity 0.8.2;
+
+import {ERC721Upgradeable} from "@openzeppelin/contracts-upgradeable/token/ERC721/ERC721Upgradeable.sol";
+import {AccessControlUpgradeable} from "@openzeppelin/contracts-upgradeable/access/AccessControlUpgradeable.sol";
+import {IMintableERC721} from "../common/interfaces/@maticnetwork/pos-portal/root/RootToken/IMintableERC721.sol";
+import {IERC721Token} from "../common/interfaces/IERC721Token.sol";
+import {IERC721Minter} from "../common/interfaces/IERC721Minter.sol";
+
+abstract contract BaseERC721 is
+    AccessControlUpgradeable,
+    ERC721Upgradeable,
+    IMintableERC721,
+    IERC721Token,
+    IERC721Minter
+{
+    uint256[50] private __gap1; // In case
```

Comment:
shouldn't it be after all the variables?

Reply:
Looking into this. There are 2 gaps (same as AvatarBase): one before and one after